src/Compilers/CSharp/Test/Semantic/Semantics/PrimaryConstructorTests.cs

```diff
@@ -16476,6 +16476,68 @@ void verify1(string source1, string source2, string source3)
             }
         }
 
+        [Fact]
+        [WorkItem("https://github.com/dotnet/roslyn/issues/67099")]
+        public void NullableWarningsForAssignment_01_NotCaptured()
+        {
+            var source = @"
+#nullable enable
+class C1(string p1, string p2)
```

Comment:
Is this fixed for records, too, as mentioned in the issue? Should we add a test?

Reply:
> Is this fixed for records, too, as mentioned in the issue? Should we add a test?

Records do not capture parameters the same way and it was not my intent to change existing behavior for records in this PR. I will make sure the issue is not closed and is properly triaged for the record scenario. Thanks for brining this up.